core/orchestrator.go

```diff
@@ -51,6 +51,18 @@ type orchestrator struct {
 	secret  []byte
 }
 
+//Format of broadcasterPrices json
```

Comment:
If it's used only in `starter.go`, then could you maybe move it there?

Reply:
Refactored and moved